documentation/general/dash-sonic-hld.md

```diff
@@ -357,13 +376,15 @@ priority                 = INT32 value  ; priority of the rule, lower the value,
 action                   = allow/deny
 terminating              = true/false   ; if true, stop processing further rules
 protocols                = list of INT ',' separated; E.g. 6-tcp, 17-udp; if not provided, match on all protocols
-src_addr                 = list of source ip prefixes ',' separated
-dst_addr                 = list of destination ip prefixes ',' separated
-src_port                 = list of range of source ports ',' separated
-dst_port                 = list of range of destination ports ',' separated
+src_tag                  = list of source tag name ',' separated; if not provided, match on all source TAGs or no TAG.
```

Comment:
what does it mean by "match on all source TAGs or no TAG" ?

Reply:
Intention is if the attribute is not provided, it is like any other ACL rule.  If not provided, wildcard match.  Match on ANY tag or NO tag (reword)?  
@Pterosaur @prsunny